Uplift/Models/Gym.swift

```diff
@@ -253,6 +253,29 @@ struct FacilityDetail {
             subfacilities = detailData.subFacilityNames.compactMap({ $0 })
         }
     }
+
+    func getEquipmentCategories() -> [EquipmentCategory] {
+       var equipmentDict = [String: [Equipment]]()
+        equipment.forEach { equipmentItem in
+            if let _ = equipmentDict[equipmentItem.equipmentType] {
```

Comment:
pref to use `equipmentDict[equipmentItem.equipmentType] != nil` instead of empty placeholder